### ISSUE TITLE
Interrupts don't Interrupt

### ISSUE THREAD

**Issue by sjstrazdus** [CalderCob/EOA-bug-tracker#135]
### Skill Name

Burglar's Addle
Champion's Clobber

### Description of the issue

The above skills are supposed to interrupt enemy inductions.  It's difficult to know for sure which skills should be interruptible in SoA because there is not animation circle/enemy induction bar like there is on retail, but going from retail behavior:

- Archer orcs and oathbreakers do a skill where they apply a poison to you that should be interruptible.
- healer/leader enemies do a heal which should be interruptible.
- Bears do an attack where they stand on their hind legs and apply a fear to you.  This should be interruptible. also on retail bears do not rotate during this induction so you can run around behind them and avoid the fear.

I've tried clobber and addle on many of these enemies and it doesn't interrupt any of them.  I can riddle orcs to get them to stop the attack, but I don't get the 'interrupted' message over their head.

Someone mentioned the red maid in another bug, but it seems to be a general problem in the entire game.

**Comment by Stressy23** [comment]
I've interrupted hillman healers and bears with addle. Rhe timing is much tighter in SoA and without instant on them you need to wait for other skills to finish before you can use them.

**Comment by sjstrazdus** [comment]
I had noticed the timing was very tight, and spent quite some time just auto attacking and being unsuccessful on any interrupts.  Inspired by Stressy23's message I went out and tried it on some southern scouts at outlaw's haven and was able to interrupt them with addle.  This was standing at range on a burglar, so not even an auto attack animation to wait for.  Perhaps that was the issue.  Bears are more frustrating to test on because they only seem to do the skill occasionally and they like to put bleeds on you.
